src/xnnpack/microparams.h

```diff
@@ -1529,6 +1538,15 @@ union xnn_f32_qu8_cvt_params {
     uint8_t output_max;
   } neonv8;
 #endif  // XNN_ARCH_ARM || XNN_ARCH_ARM64
+#if XNN_ENABLE_RISCV_VECTOR && XNN_ARCH_RISCV
```

Comment:
After you update your branch from master, please do not add new architecture-specific params here, you shouldn't need them in this case.